compiler/rustc_ast/src/ast.rs

```diff
@@ -1194,7 +1194,7 @@ impl Expr {
         }
     }
 
-    /// Returns an expression with (when possible) *one* outter brace removed
+    /// Returns an expression with (when possible) *one* outter outter brace removed
```

Comment:
Is it for testing CI or just a typo?

Reply:
This is testing typo from `[dont merge] add typo` commit.

Reply:
Perfect, resolving discussion then, thanks!